handler.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const request = require('superagent')
+const async = require('async')
+const config = require(`./config/production.json`)
```

Comment:
IMO, contributors should duplicate sample.json -> production.json and put in their own keys. This is open source anyway.